tests/phpunit/tests/category.php

```diff
@@ -9,11 +9,6 @@
  */
 class Tests_Category extends WP_UnitTestCase {
 
-	function tearDown() {
-		_unregister_taxonomy( 'test_tax_cat' );
```

Comment:
Similar remark as before:
Taxonopmies **are** being (conditionally) reset in `WP_UnitTestCase_Base::reset_taxonomies()` which is called in `WP_UnitTestCase_Base::setUp()`, but that is _unstable_.

Reply:
What's the reason that's unstable? The `WP_RUN_CORE_TESTS` constant is the means with which the test suite determines that this is a core test run as opposed to a test run from a plugin or theme.